core/src/toga/screen.py

```diff
@@ -0,0 +1,11 @@
+class Screen:
+    def __init__(self, _impl):
+        self._impl = _impl
+
+    @property
+    def name(self):
+        return self._impl.get_name()
+
+    @property
+    def origin(self):
+        return self._impl.get_origin()
```

Comment:
It seems like the other obvious thing you'd want to do with a screen is know how big it is.

Reply:
Added a `Screen.size` property.